scripts/reset-test-db.sh

```diff
@@ -48,13 +48,11 @@ else
     rm -rf $DB_CACHE_DIR && mkdir -p $DB_CACHE_DIR
 
     # Re-run migrations on the test database
-    ./manage.py lms --settings bok_choy syncdb --traceback --noinput
-    ./manage.py cms --settings bok_choy syncdb --traceback --noinput
     ./manage.py lms --settings bok_choy migrate --traceback --noinput
     ./manage.py cms --settings bok_choy migrate --traceback --noinput
 
     # Dump the schema and data to the cache
-    ./manage.py lms --settings bok_choy dumpdata > $DB_CACHE_DIR/bok_choy_data.json
-    mysqldump -u root --no-data --skip-comments --skip-dump-date edxtest > $DB_CACHE_DIR/bok_choy_schema.sql
+    # ./manage.py lms --settings bok_choy dumpdata > $DB_CACHE_DIR/bok_choy_data.json
+    # mysqldump -u root --no-data --skip-comments --skip-dump-date edxtest > $DB_CACHE_DIR/bok_choy_schema.sql
```

Comment:
I have created a ticket to re-enable this once our migrations have stabilised: https://openedx.atlassian.net/browse/TNL-3698